sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -718,7 +722,9 @@ object ParquetFileFormat extends Logging {
   }
 
   /**
-   * ParquetFileFormat.prepareWrite calls this function to initialize `redirectParquetLogsViaSLF4J`.
+   * The `ParquetFileFormat` constructor calls this method to ensure that Parquet library log
+   * output is redirected through the SLF4J JUL bridge handler. This method is a no-op because we
+   * only require that `redirectParquetLogsViaSLF4J` is initialized. This accomplishes that task.
    */
-  def redirectParquetLogs(): Unit = {}
+  private def ensureParquetLogRedirection(): Unit = {}
```

Comment:
OK this is better, though on a second look, it seems less indirect to either:

a) use a `lazy val` above and simply do anything with that reference that compiles, like log it or something. It didn't before because just referencing a field and doing nothing else will cause a warning, or

b) make it a method that executes once with a simple `AtomicBoolean` guard or boolean + synchronization

I suppose I'm wondering whether even this mechanism is guaranteed to work.